lib/rails_values/display_name_email_address.rb

```diff
@@ -5,7 +5,7 @@ class DisplayNameEmailAddress < EmailAddress
     include Comparable
     include WholeValueConcern
 
-    delegate :format, to: :mail_address
+    delegate :format, :address, to: :mail_address
```

Comment:
while email_address class implicitly exposes the address component of an email via `to_str`, I think that will make my use ambiguous in my PR where I create TenantContacts when saving email marketing senders that contain a display name.